wandb/apis/public/reports.py

```diff
@@ -69,24 +84,30 @@ def length(self):
 
     @property
     def more(self):
+        """Returns `True` if there are more files to fetch. Returns
+        `False` if there are no more files to fetch.
+        """
```

Comment:
```suggestion
        """Returns whether there are more files to fetch."""
```

+ annotate the return type as `bool` (this might cause type-checking errors depending on the type of `self.last_response`, in which case we'll just need to surround that expression by `bool(...)`)